framework/src/controller/controller.js

```diff
@@ -93,54 +93,65 @@ module.exports = class Controller {
 		);
 	}
 
-	async _loadModules() {
-		return Promise.each(Object.keys(this.modules), m =>
-			this._loadInMemoryModule(m)
+	async _loadModules(modules) {
+		return Promise.each(Object.keys(modules), alias =>
```

Comment:
I think we should avoid using bluebird methods since it's just making harder to migrate using native Promises.